contributing/documentation.md

```diff
@@ -179,15 +179,32 @@ You can let a code sample have a preview or a link to open the code sample in th
   </code>
 </pre>
 
-An inline preview is only possible if the code does not contain any head elements.
-For email content with resource links use the placeholder `{{server_for_email}}`.
+Use the `preview` attribute to define how the preview is generated:
+- **none**: No preview will be generated
+
+- **inline**: The example is included directly in the page above the source code display.
```

Comment:
rephrase: "The example is included directly in the page above the source code display"

to 

"The example preview is displayed above the source code"